lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -51,7 +56,14 @@
   </div>
  % endif
 <article class="course${mode_class}">
-  <% course_target = reverse('info', args=[unicode(course_overview.id)]) %>
+  <%
+    course_locator = CourseLocator.from_string(unicode(course_overview.id))
```

Comment:
Could we move most of this logic to the function, so that it only receives the course_id as parameter. This way this piece of logic would also be unit-tested